### PR TITLE
Add ledger-tool bigtable upload loop

### PR DESCRIPTION
**Pull request by CriesofCarrots**
#### Problem
https://github.com/solana-labs/solana/pull/24716 added a configurable limit to the number of blocks that `solana_ledger::bigtable_upload::upload_confirmed_blocks` checks at a time. However, it neglected to update `solana-ledger-tool`.

#### Summary of Changes
Add a loop to `solana-ledger-tool bigtable upload` so that the upload service iterates through the entire requested range, instead of stopping at the first limit.

Fixes #26023

### CODE REVIEW

**Comment by CriesofCarrots** [comment]
cc @artburkart

File: ledger-tool/src/bigtable.rs [hunk 1]
```diff
@@ -49,19 +49,27 @@ async fn upload(
...
-        bigtable,
-        starting_slot,
-        ending_slot,
-        config,
-        Arc::new(AtomicBool::new(false)),
-    )
-    .await
-    .map(|last_slot_uploaded| {
+    loop {
+        if starting_slot > 0 && starting_slot == last_slot_uploaded {
```
Comment (artburkart) [inline ledger-tool/src/bigtable.rs:56]:
```suggestion
        if starting_slot >= 0 && starting_slot == last_slot_uploaded {
```
Is there a hard requirement that `starting_slot` not be 0?

Comment (CriesofCarrots) [reply ledger-tool/src/bigtable.rs:56]: 0 is a valid `starting_slot`; that's why we don't want to break out of the loop there.

Comment (artburkart) [reply ledger-tool/src/bigtable.rs:56]: Oh duh. 😅

**Pull request merged**